### DISCUSSION TITLE
Dax Studio does not open

### DISCUSSION THREAD

**Discussion by NeoArt3** [DaxStudio/DaxStudio#754] [General]
I can't open DaxStudio from Excell. But previous it work always good. Delete and reinstall DaxStudio is not help. 
And I can't open portable version too.
Version =  v2.16.2.632
    DaxStudio.exe Version =  v2.16.2.632

Checking Operating System
=======================

FATAL ERROR:
Неправильное пространство имен 
   в System.Management.ThreadDispatch.Start()
   в System.Management.ManagementScope.Initialize()
   в System.Management.ManagementObjectSearcher.Initialize()
   в System.Management.ManagementObjectSearcher.Get()
   в DaxStudio.CheckerApp.SystemInfo.GetOSInfo()
   в DaxStudio.CheckerApp.SystemInfo.OutputOSInfo(RichTextBox output)
   в DaxStudio.CheckerApp.Checker.CheckOSInfo()
   в DaxStudio.CheckerApp.MainWindow.OnLoaded(Object sender, RoutedEventArgs e)

**Comment by dgosbell** [comment]
That looks like your machine might have a corruption in the WMI information similar to this issue https://github.com/DaxStudio/DaxStudio/issues/750 can you try the preview build linked in that issue and see if that works for you?

**Reply by NeoArt3** [reply]
Thank you! #750 works

**Comment by LucuisSulla** [comment]
Hi,  I have an issue that is a duplicate of Issue #489.  I see from the issue thread that the issue has been closed as the IT department of the person raising the ticket is reported to have fix this issue.  Here is a quote from the issue:   "Our IT department has resolved this by adjusting the anti-virus policy. This has resolved the issue in my case. DAX Studio now opens without error."

I would like to contact the issue raiser "TedSkomsky" to see if his IT department can specifiy what policy change they made to fix this issue.  

Note I am unfamiliar with GitHub policy on direct contact, but when I clicked on the name TedSkomsky, I did not see a method to email directly.

Regards,

K

**Reply by dgosbell** [reply]
If TedSkomsky has not enabled direct contact then the best option is to reply to that other issue and use an `@TedSkomsky` mention to tag him. He should then get a notification depending on how he configured his github account. 

But if you are getting access denied errors your IT department should have access to logs to tell them what is being blocked and why. At a guess I would say that they probably added daxstudio.exe to the AV whitelist. The other possibility is that they switched from a "Current User" to an 'All Users" installation. The "All Users" install puts DAX Studio in the "program files" folder which is typically trusted more by AV scanners since it requires admin rights to install there
